setup.cfg

```
@@ -0,0 +1,11 @@
+[flake8]
+ignore = E501, W503, E203, C901, N806
```

```suggestion
ignore = W503, E203
```
요렇게 하고 lint 에러가 나는 경우 해당 코드를 수정하는걸로 하시죠